Nutanix Karbon CNCF conformance results submission for 1.16

basu-ntnx opened the pull request:
### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [ ] If this is a new entry, have you submitted a signed [participation form](https://github.com/cncf/k8s-conformance/tree/master/participation-form)?
* [* ] Did you include the product/project logo in SVG, EPS or AI format?  
* [ *] Does your logo clearly state the name of the product/project and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [ ] If your product/project is open source, did you include the `repo_url`?
* [*] Did you copy and paste the [installation and configuration instructions](https://github.com/cncf/k8s-conformance/blob/master/faq.md#can-i-provide-a-link-to-the-installation-directions) into the README.md file in addition to linking to them?

taylorwaggoner commented:
Hello, thanks for submitting!  A couple things here.  First, we have a participation agreement on file for this product, so I think you can remove that file from the PR.  

We've also recently been tightening our README requirements. Please review [these guidelines](https://github.com/cncf/k8s-conformance/blob/master/faq.md#can-i-provide-a-link-to-the-installation-directions) and paste all installation/configuration instructions into your README, in addition to the URLs you already included.  Let me know if you have questions!

Thanks!

basu-ntnx commented:
@taylorwaggoner : Thank you for the comments. I have removed the participation agreement and updated relevant parts of the README. Please review. Thank you.

taylorwaggoner commented:
@basu-ntnx thanks for removing the file.  For the README, could you please paste all instructions into the file rather than directing people to the URL you linked?  We're trying to minimize link rot by having all instructions pasted into your pull request.  Thanks!
